ml-dhs-aws/src/test/java/com/marklogic/client/dhs/test/TestJobRunner.java

```diff
@@ -68,6 +68,7 @@ public void testRun() throws IOException {
          testBeforeJobDocument(jobControlObj);
          testAfterJobDocument(jobControlObj);
          testJsonDataStructure(jobControlObj);
+         testLoggerStatus(jobRunner.getStatus());
```

Comment:
Instead, should the test create a TestJobLogger class that implements JobLogger, initialize jobRunner with an instance of the class, and make assertions about the status objects passed to the TestJobLogger?